R/table.R

```diff
@@ -242,6 +262,177 @@ setMethod("dbWriteTable", c("MariaDBConnection", "character", "character"),
     invisible(TRUE)
   }
 )
+#' @export
+#' @rdname mariadb-tables
+#' @details
+#' When using `load_data_local_infile = TRUE` in [dbConnect()],
+#' pass `safe = FALSE` to `dbAppendTable()` to avoid transactions.
+#' Because `LOAD DATA INFILE` is used internally, this means that
+#' rows violating primary key constraints are now silently ignored.
+#' @importFrom utils write.table
+setMethod("dbAppendTable", "MariaDBConnection",
+  function(conn, name, value, ..., row.names = NULL) {
+    if (!is.null(row.names)) {
+      stop("Can't pass `row.names` to `dbAppendTable()`", call. = FALSE)
+    }
+    stopifnot(is.character(name), length(name) == 1)
```

Comment:
This check breaks the usage of `DBI::Id(schema = "foo", table = "bar")`. Is it intended?